server/service/src/sync/remote_data_synchroniser.rs

```diff
@@ -47,6 +53,26 @@ impl RemoteDataSynchroniser {
         Ok(())
     }
 
+    /// Request site info and persist it
+    pub(crate) async fn request_and_set_site_info(
+        &self,
+        connection: &StorageConnection,
+    ) -> Result<(), RequestAndSetSiteInfoError> {
+        info!("Requesting site info");
+        let site_info = self
+            .sync_api_v5
+            .get_site_info()
+            .await
+            .map_err(RequestAndSetSiteInfoError::RequestSiteInfoError)?;
+
+        let remote_sync_state = RemoteSyncState::new(&connection);
+        remote_sync_state.set_site_uuid(site_info.id).map_err(RequestAndSetSiteInfoError::SetSiteInfoError)?;
```

Comment:
Did you do this change in vs code ? It's not formatted

Reply:
Yup I did. I just turned on `format on save` now 🙃

Reply:
Cool, good to go on my end once format is reinstated, tests passed

Reply:
https://github.com/openmsupply/open-msupply/pull/388/commits/3be125b8358ef06077a0ae207e9bfcfc6d190bc1